tests/test_config.py

```diff
@@ -133,18 +133,19 @@ def test_throttler(self):
 
     def test_for_unexpected_config_entries(self):
         with self.assertRaises(Exception):
-            _ = Config({"unexpected":"value"}, validate=True)
+            _ = Config({'unexpected': 'value'}, validate=True)
+            assert _
```

Comment:
Can you do this same as you did for other cases? i.e. just call `Config({"unexpected":"value"}, validate=True)`

Reply:
what is the purpose of adding assert, since it's never expected to be reached?